pynetbox/core/query.py

```diff
@@ -409,5 +409,5 @@ def get_count(self, *args, **kwargs):
         """
 
         if not hasattr(self, "count"):
-            self.count = self._make_call(add_params={"limit": 1})["count"]
+            self.count = self._make_call(add_params={"limit": 1, "brief": 1})["count"]
```

Comment:
This is just a minor nitpick, but I would prefer this to be set to `True`. Although, python evaluates `1` the same but a true boolean value conveys the meaning and intention much better.

Reply:
Hi,
NetBox documentation specifically refers to brief=1
https://demo.netbox.dev/static/docs/rest-api/overview/